client/src/components/Inputs.tsx

```diff
@@ -65,6 +65,72 @@ export function PasswordInput({
   );
 }
 
+export function CurrentPasswordInput({
+  inputRef,
+}: {
+  inputRef?: RefObject<HTMLInputElement>;
+}) {
+  return (
+    <div className="row margin-bottom--md">
+      <label htmlFor="password" className="margin-right--sm">
+        Current Password
+      </label>
+      <input
+        ref={inputRef}
+        type="password"
+        name="current-password"
+        autoComplete="current-password"
+        id="password"
```

Comment:
An id of `password` collide with other rendered components using the same id; we could possibly forgo the id attribute entirely here and elsewhere, or make it more specific to this variant

```suggestion
        id="current-password"
```